storage/entities/account.js

```diff
@@ -325,22 +325,30 @@ class Account extends BaseEntity {
 	/**
 	 * Create account object
 	 *
-	 * @param {Object} data
-	 * @param {Object} [options]
-	 * @param {Object} tx - Transaction object
+	 * @param {Object|Array.<Object>} data
+	 * @param {Object} [_options]
+	 * @param {Object} [tx] - Transaction object
 	 * @return {*}
 	 */
 	create(data, _options, tx) {
-		const objectData = _.defaults(data, defaultCreateValues);
-		const createSet = this.getValuesSet(objectData);
-		const attributes = Object.keys(data)
+		let values;
+
+		if (Array.isArray(data)) {
+			values = data;
+		} else if (typeof data === 'object') {
+			values = [data];
+		}
+
+		values = values.map(v => _.defaults(v, defaultCreateValues));
+		const createSet = this.getValuesSet(values);
+		const attributes = Object.keys(values[0])
```

Comment:
I find it risky => `Object.keys(values[0])`
There is no validation for `data` meaning you can't guarantee it will only contain the `defaultCreateValues`. Also, if the `data` attributes have different order in different objects in the array, it will error.

I would add a validation for `data` and then order its keys to match `defaultCreateValues`.

Reply:
I don't think order of keys will have any impact. And wince we are merging all keys to default values, so all objects must have same keys.

Reply:
You're trying to construct this, right?
```
INSERT INTO table (id, name)
VALUES  ('1', 'name1'),
        ('2', 'name2'),
        ('3','name3')
```
If so, the order matter.

Try `storage.entities.Account.create([{ address: '123L', username: 'primeiro' }, { username: 'segundo', address: '321L' }])` and you gonna see what I'm talking about.

Reply:
I do agree with @lsilvs here it could cause psql rejections in the best case or invalid data in the worst case (if types of data match)